openapi.yaml

```diff
@@ -6585,6 +6585,67 @@ paths:
       - lang: CLI
         source: >
           linode-cli lke pool-delete 12345 456
+  /lke/clusters/{clusterId}/api-endpoints:
+    parameters:
+    - name: clusterId
+      in: path
+      description: ID of the Kubernetes cluster to look up.
+      required: true
+      schema:
+        type: integer
+    x-linode-cli-command: lke
+    get:
+      operationId: getLKEClusterAPIEndpoints
+      x-linode-cli-action: api-endpoints-list
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - lke:read_only
+      tags:
+      - Linode Kubernetes Engine (LKE)
+      servers:
+      - url: https://api.linode.com/v4beta
+      summary: List Kubernetes API Endpoints
+      description: >
+        List the Kubernetes API server endpoints for this cluster. Please note that it often takes
+        2-5 minutes before the endpoint is ready after first [creating a new cluster](/api/v4/lke-clusters/#post).
```

Comment:
/v4beta

Reply:
If this is in regard to the `creating a new cluster` link--I think that the `v4` in this URL is not reflective of the v4 vs v4beta for the endpoint URL

Reply:
ah it's just an anchor I see.